blender/arm/make_renderpath.py

```diff
@@ -178,10 +178,8 @@ def build():
             assets.add(assets_path + 'clouds_map.png')
             assets.add_embedded_data('clouds_map.png')
 
-    if rpdat.rp_renderer == 'Deferred':
-        assets.add_shader_pass('copy_pass')
-    else:
-        assets.add_khafile_def('rp_forward')
+    assets.add_shader_pass('copy_pass')
+    assets.add_khafile_def('rp_forward')
```

Comment:
Aren't you adding the `rp_forward` define here even if deferred is used? This doesn't look correct.

Reply:
True, I'm debugging this ATM.
I'm issue with forward, even while all refraction's related clauses are deactivated in the renderpath:

Trace: TypeError: Cannot read property 'isCubeMap' of undefined
    at iron_RenderPath.setTarget (<anonymous>:4668:21)